contributing.md

```diff
@@ -175,3 +175,28 @@ You may then view `output.log` in VSCode. A typical log line is then:
 ```log
 [14:37:36.517] DEBUG <engine.go:114>: 0x111A00  <  {"engine":"0x111A00","message":"Constructed Engine"}
 ```
+
+### Start RPC servers with Docker
+
+To spin up a docker image with 3 rpc servers and channels pre-populated, run the following:
+
+1. `make docker/build`
```

Comment:
this instruction seems fine, but also a naked `make` (typo) seemed to do the trick for me as well.